asynctnt/instance.py

```diff
@@ -35,19 +36,19 @@ def __init__(self, tnt, on_exit):
         super().__init__()
         self._tnt = tnt
         self._on_exit = on_exit
-        self._transport = None
+        self._connection_transport = None
```

Comment:
This is what I've been referring earlier - it does not make sense to call this variable a `connection_transport`

Reply:
Reworked, named a new one as `_parameter_transport`.